config.toml

```diff
@@ -25,6 +25,51 @@ home = [ "HTML", "RSS", "JSON"]
   themeVariant = "red"
   disableInlineCopyToClipBoard = "false"
 
+## Use those parameters with {{< param "param_name" >}}
+
+#  deployment = "agnosticd"
+#  external_domain = "<GUID>.${LABID}.opentlc.com"
+#  internal_domain = "<GUID>.internal"
+#  internal_control = "bastion.<GUID>.internal"
+#  internal_host1 = "node1.<GUID>.internal"
+#  internal_host2 = "node2.<GUID>.internal"
+#  internal_isonode = "worker1.emea.<GUID>.internal"
+#  internal_tower1 = "tower1.<GUID>.internal"
+#  internal_tower2 = "tower2.<GUID>.internal"
+#  internal_tower3 = "tower3.<GUID>.internal"
+#  internal_towerdb = "support1.<GUID>.internal"
+#  external_tower = "tower.<GUID>.${LABID}.opentlc.com"
+#  control_prompt = "ec2-user@bastion"
+#  ansible_prompt = "root@bastion"
+#  pre_awx_prompt = "TBD"
+#  awx_prompt = "TBD"
+#  content_git_uri = "structured-content.git"
+
+  deployment = "linklight"
+  external_domain = "${LABID}.events.opentlc.com"
+  internal_domain = "${LABID}.internal"
+  # control or bastion host
+  internal_control = "student${N}-ansible.${LABID}.internal"
+  # we don't have a load balancer, hence tower1 is same as tower
+  external_tower = "student${N}-ansible.${LABID}.events.opentlc.com"
+  external_tower1 = "student${N}-ansible.${LABID}.events.opentlc.com"
+  external_tower2 = "student${N}-towernode2.${LABID}.events.opentlc.com"
+  external_tower3 = "student${N}-towernode3.${LABID}.events.opentlc.com"
+  external_code = "student${N}-code.${LABID}.events.opentlc.com"
+  internal_tower1 = "student${N}-ansible.${LABID}.internal"
+  internal_tower2 = "student${N}-towernode2.${LABID}.internal"
+  internal_tower3 = "student${N}-towernode3.${LABID}.internal"
+  internal_towerdb = "student${N}-towerdb.${LABID}.internal"
+  internal_toweriso = "student${N}-isonode.${LABID}.internal"
+  control_prompt = "student@ansible"
```

Comment:
this should IMHO also be student${N}@ansible .etc.

Reply:
it was like this before and I haven't put variables everywhere, so let's keep like this for now.